lte/gateway/release/magma-postinst-bazel

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+# Copyright 2020 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Delete OVS bridge on boot
+sed -i "s/.*OVS_CTL_OPTS.*/OVS_CTL_OPTS='--delete-bridges'/" /etc/default/openvswitch-switch
+
+# Create /var/core directory
+mkdir -p /var/core
+
+value=`cat /usr/local/share/magma/commit_hash`
+if grep -q "COMMIT_HASH" /etc/environment
+then
+    sudo sed -i -e "s/^COMMIT_HASH.*/$value/" /etc/environment
+else
+    echo "$value" | sudo tee -a /etc/environment
+fi
+
+# Set magmad service to start on boot
+systemctl enable -f magma@magmad.service
+
+# Installation of the redis-server Debian package has a post-install
+# script that starts a redis-server process. We kill and disable this
+# process, allowing us to manage the process using magmad and our own
+# systemd files.
+systemctl stop redis-server
+systemctl disable redis-server
+
+# Dnsmasq also starts up post-install.
+systemctl stop dnsmasq
+systemctl disable dnsmasq
+
+# Lighttpd also starts up post-install.
+systemctl stop lighttpd
+systemctl disable lighttpd
+
```

Comment:
I guess it's intentional that you didn't add the `Copy over ipfix patch` part from the other postinst script?

Reply:
yes, with bazel the ryu patches are applied in `bazel/python_repositories.bzl:24`++ and the patched ryu is packed into the magma.deb.